### ISSUE TITLE
Name your decision forest models

### ISSUE THREAD

**Issue by alex-ashford-ovo** [tensorflow/decision-forests#35]
I would like to be able to explicitly name my model. I've seen that the models have a name attribute but it does not appear to be possible to set this manually.

I've tried:
```
setattr(model, 'name', 'my_cool_model')
```

And:
```
tfdf.keras.RandomForestModel(name = 'my_cool_model')
```

**Comment by achoum** [comment]
The error indicates that the attribute `name` is already used. You can use another name e.g. `model.model_name = 'my_cool_model'` will work.

Alternatively, you can store this model in a new class e.g. `ModelAndMetaData`. This way, there is no risk to override an existing attribute and possibly perturbe the internal logic of the model.

**Comment by alex-ashford-ovo** [comment]
Is this not a functionality gap as you can do this with the standard keras API?

**Comment by achoum** [comment]
You are right. I did not realize this was part of the Keras API :).

The change has been submitted, and it will be part of the next TF-DF release (v0.1.8).

**Comment by alex-ashford-ovo** [comment]
Thank you :)

**Comment by achoum** [comment]
The change was published in TF-DF 0.1.8.
